.travis.yml

```diff
@@ -1,7 +1,37 @@
 language: node_js
 node_js:
-- '0.10'
+- '6'
+before_install:
+- wget https://s3-us-west-2.amazonaws.com/cdn.thegrid.io/caliper/libvips/libccv-0.1.1.tar.gz
+- mkdir $HOME/libccv
+- echo $HOME/libccv
+- tar -xvzf libccv-0.1.1.tar.gz -C $HOME/libccv
 before_script:
 - npm install -g grunt-cli
-after_success:
-- grunt gh-pages
+deploy:
+  provider: script
+  skip_cleanup: true
+  script: grunt gh-pages
+  on:
+    tags: true
+    repo: noflo/noflo-browser
+env:
+  matrix:
+  - CXX=g++-4.8
+  global:
+    secure: mXwYeFfEjRsB31W5sorAjBXNPc2ZqV1IabgGv4fx8nmLGJ2KVCg7FRtJxPkshCxUMbn69S5JR12HcKWsFNY4uF0un7On70ggcq1+TpRFh8VkDr2KuRTsxW0NuZE3SO/xx7fxMQ98woHnW1FqLrQMhlZLDdW+OSVrx8/sJ4Xql3oZgt/n3VGaIQuJ6D9cvx/TT8WqxF+cEunm1T0eHpjtVTEKQHFDsOtztWJR/tHDrxpP0J8OrpEXtdFGXXbS8od4wCSJ0qmZXPfuD7cQgwwMLa3FnHxC5B4SOcjOagr8cr269BTSdV8CN5udVb8g2vuvF33T/Q7KFZKd/G9ue1w+7unyAG6MZT6cN3uzL5JwsAsU9C8fRF94yD7Q3dfkI+UMssf0kt3zfBU/XgvJJQ4mzoaouGcgdB2/jn0PbQrECUfILhG4/c7Y/6hQUjxJelc2hunXkY7QwOuRi4xNGsTgEZwBT7pZmPXRwOABZwdo/4WU6y4vTKINyX4f4di/bIOtewoTT+tYwab9A7xqka8SUTyc1FHuQyzAww9BK4TGjaWxKP7dYPmLdKWzGPUhEKwzsNBIKvB6g5vIULb6T7x4/UXOlI7hKnM1rUOpiYLto4RV4LJlIksUKpaC8z/dtmKko3qThXWNJc9tF6trCEUjV03xXfgAh1ohsV0/euVwYLk=
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - libpng-dev
```

Comment:
Does not change the point. These dependencies are awful and should be completely uneccesary for a *browser* project. This will mean lots of people on Mac won't be able to build `noflo-browser`. Basically no-one on Windows will be able. Even some peopl Linux people will struggle.